pinot-core/src/main/java/org/apache/pinot/core/minion/rollup/MergeRollupSegmentConverter.java

```diff
@@ -45,16 +49,16 @@
   private TableConfig _tableConfig;
   private String _tableName;
   private String _segmentName;
-  private MergeType _mergeType;
+  private CollectorType _collectorType;
```

Comment:
I think that we can just revert these changes back. Your separate PR for migrating from  to `SegmentProcessingFramework` will deprecate the usage of `MergeRollupSegmentConverter`

Reply:
Yes, this will be deprecated, I'll revert these changes back.